### ISSUE TITLE
Wails Dev Fails with option -r not recognized

### ISSUE THREAD

**Issue by GritWins** [wailsapp/wails#3440]
### Description

After Installing Wails in Mac and while running the `wails dev` command  I am getting the following error

```
Wails CLI v2.8.0

Executing: go mod tidy
  • Generating bindings:
  ERROR   option -r not recognized

          usage: xattr [-slz] file [file ...]
                 xattr -p [-slz] attr_name file [file ...]
                 xattr -w [-sz] attr_name attr_value file [file ...]
                 xattr -d [-s] attr_name file [file ...]
                 xattr -c [-s] file [file ...]

          The first form lists the names of all xattrs on the given file(s).
          The second form (-p) prints the value of the xattr attr_name.
          The third form (-w) sets the value of the xattr attr_name to attr_value.
          The fourth form (-d) deletes the xattr attr_name.
          The fifth form (-c) deletes (clears) all xattrs.

          options:
            -h: print this help
            -s: act on symbolic links themselves rather than their targets
            -l: print long format (attr_name: attr_value)
            -z: compress or decompress (if compressed) attribute value in zip format


          exit status 64
```

I tried the `wails doctor` command and it shows everything OK

```
github_demo|⇒ wails doctor


          Wails Doctor



# Wails
Version | v2.8.0

# System
┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
| OS           | MacOS                                                                                                                                                                                                                             |
| Version      | 13.0.1                                                                                                                                                                                                                            |
| ID           | 22A400                                                                                                                                                                                                                            |
| Go Version   | go1.20.3                                                                                                                                                                                                                          |
| Platform     | darwin                                                                                                                                                                                                                            |
| Architecture | amd64                                                                                                                                                                                                                             |
| CPU          | Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz                                                                                                                                                                                          |
| GPU          | Chipset Model: Intel UHD Graphics 630 Type: GPU Bus: Built-In VRAM (Dynamic, Max): 1536 MB Vendor: Intel Device ID: 0x3e9b Revision ID: 0x0000 Automatic Graphics Switching: Supported gMux Version: 5.0.0 Metal Support: Metal 3 |
| Memory       | 16GB                                                                                                                                                                                                                              |
└──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

# Dependencies
┌─────────────────────────────────────────────────────────────────────────┐
| Dependency                | Package Name | Status    | Version          |
| Xcode command line tools  | N/A          | Installed | 2396             |
| Nodejs                    | N/A          | Installed | 18.16.0          |
| npm                       | N/A          | Installed | 9.5.1            |
| *Xcode                    | N/A          | Installed | 14.3.1 (14E300c) |
| *upx                      | N/A          | Available |                  |
| *nsis                     | N/A          | Available |                  |
└──────────────────────── * - Optional Dependency ────────────────────────┘

# Diagnosis
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/
  - nsis : More info at https://wails.io/docs/guides/windows-installer/

 SUCCESS  Your system is ready for Wails development!

 ♥   If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony
```          


### To Reproduce

1. Install Wails 
2. Run `wails init -n appname -t vue`
3. cd appname
4. Run `wails dev`

### Expected behaviour

`wails dev` should not throw this error and open a desktop sample app

### Screenshots

<img width="924" alt="Screenshot 2024-04-27 at 9 58 34 PM" src="https://github.com/wailsapp/wails/assets/82824111/0cecbfab-d60a-49a4-bc3e-ff10def8492e">


### Attempted Fixes

_No response_

### System Details

```shell
# Wails
Version | v2.8.0

# System
┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
| OS           | MacOS                                                                                                                                                                                                                             |
| Version      | 13.0.1                                                                                                                                                                                                                            |
| ID           | 22A400                                                                                                                                                                                                                            |
| Go Version   | go1.20.3                                                                                                                                                                                                                          |
| Platform     | darwin                                                                                                                                                                                                                            |
| Architecture | amd64                                                                                                                                                                                                                             |
| CPU          | Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz                                                                                                                                                                                          |
| GPU          | Chipset Model: Intel UHD Graphics 630 Type: GPU Bus: Built-In VRAM (Dynamic, Max): 1536 MB Vendor: Intel Device ID: 0x3e9b Revision ID: 0x0000 Automatic Graphics Switching: Supported gMux Version: 5.0.0 Metal Support: Metal 3 |
| Memory       | 16GB                                                                                                                                                                                                                              |
└──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

# Dependencies
┌─────────────────────────────────────────────────────────────────────────┐
| Dependency                | Package Name | Status    | Version          |
| Xcode command line tools  | N/A          | Installed | 2396             |
| Nodejs                    | N/A          | Installed | 18.16.0          |
| npm                       | N/A          | Installed | 9.5.1            |
| *Xcode                    | N/A          | Installed | 14.3.1 (14E300c) |
| *upx                      | N/A          | Available |                  |
| *nsis                     | N/A          | Available |                  |
└──────────────────────── * - Optional Dependency ────────────────────────┘

# Diagnosis
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/
  - nsis : More info at https://wails.io/docs/guides/windows-installer/

 SUCCESS  Your system is ready for Wails development!

 ♥   If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony
```
```


### Additional context

_No response_

**Comment by SaravEasy** [comment]
This was due to the`xattr` location being wrong in the $PATH and I changed it to solve it.

**Comment by leaanthony** [comment]
This was fixed in v2.8.1 👍
